src/dialects/postgres/query-interface.d.ts

```diff
@@ -0,0 +1,9 @@
+import type { Sequelize } from '../../sequelize.js';
+import { QueryInterface } from '../abstract/query-interface.js';
+import type { PostgresQueryGenerator } from './query-generator.js';
+
+export class PostgresQueryInterface extends QueryInterface {
+  queryGenerator: PostgresQueryGenerator;
+
+  constructor(sequelize: Sequelize, queryGenerator: PostgresQueryGenerator);
```

Comment:
I propose to replicate this change in the other query-interface stubs

Reply:
Yeah, I noticed that as well when I looked at it again this morning. Will update it later today